nexus/tests/integration_tests/disks.rs

```diff
@@ -1349,6 +1347,11 @@ const ALL_METRICS: [&'static str; 6] =
 
 #[nexus_test]
 async fn test_disk_metrics(cptestctx: &ControlPlaneTestContext) {
+    // Record the current start time.  When fetching data points, we never need
+    // to look at times before this timestamp, since all the events we're
+    // interested in happen after this test starts.
```

Comment:
As mentioned in chats: This will work for "metrics on a single disk", but might not work as well for "system-wide resources", which are made during the populate step.

Should be fine for this test, but caused me issues with the instance metrics test, where the sample "cpu_count = 0" was taken before the test started.